Frontend/CO.CDP.OrganisationApp.Tests/Registration/OrganisationIdentificationTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using CO.CDP.OrganisationApp.Pages.Registration;
+using CO.CDP.OrganisationApp.Tests.Common;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace CO.CDP.OrganisationApp.Tests.Registration
+{
+    public class OrganisationIdentificationModelTests
+    {
+        private readonly OrganisationIdentificationModel _pageModel;
+        private readonly Session _session;
+        private readonly DefaultHttpContext _httpContext;
+        private readonly Mock<ILogger<OrganisationIdentificationModel>> _loggerMock;
+
+        public OrganisationIdentificationModelTests()
+        {
+            _loggerMock = new Mock<ILogger<OrganisationIdentificationModel>>();
+
+            _httpContext = new DefaultHttpContext
```

Comment:
Should mock ISession rather than creating real object

Reply:
Thanks @dharmverma I have mocked ISession now. Updated and refactored the whole code.